src/ott/geometry/costs.py

```diff
@@ -347,6 +347,34 @@ def tree_unflatten(cls, aux_data, children):  # noqa: D102
     return cls(*aux_data)
 
 
+@jtu.register_pytree_node_class
+class EuclideanP(TICost):
```

Comment:
Please add to the docs.

Reply:
ok. Don't want to surprise the user because there's no Legendre transform, but can add.